app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,51 @@
+  <string name="wont_detect_code_from_this_notif">Von dieser Benachrichtigung werden keine Codes mehr detektiert.</string>
+  <string name="copied">Kopiert</string>
+  <string name="ignored_list">Ignoriert Liste</string>
+  <string name="open_notification_permission">Benachrichtigungseinstellungen öffnen</string>
+  <string name="send_test_notification">Testbenachrichtigung senden</string>
+  <string name="auto_copy">Automatisch Kopieren</string>
+  <string name="delete">Löschen</string>
+  <string name="label_back">Zurück</string>
+  <string name="HOME_ROUTE">Home</string>
+  <string name="IGNORED_LIST_ROUTE">Ignorieren Liste</string>
+  <string name="list_is_empty">Liste ist leer</string>
+  <string name="ignored_list_description">Wenn du den Ignorieren Knopf in der \"Code gefunden\"-Benachrichtigung drückst, wird diese Art Code nicht mehr detektiert und erscheint hier.</string>
+  <string name="copy">Kopieren</string>
+  <string name="ignore">Ignorieren</string>
+  <string name="PERMISSION_ROUTE">Berechtigungen</string>
+  <string name="permissions_desc">Um sicherzustellen, dass die App jederzeit Codes korrekt detektieren kann, musst du die folgenden Berechtigungen erteilen:</string>
+  <string name="open">Öffnen</string>
+  <string name="permission_todo_post_notifications">1. Benachrichtigungen senden</string>
+  <string name="permission_todo_read_notifications">2. Benachrichtigungen lesen</string>
+  <string name="grant">Erlauben</string>
+  <string name="permission_todo_remain_open">3. Offen bleiben</string>
+  <string name="perm_extra_desc">Auf manchen Geräten wie Xioami, Huawei, etc. musst du, um der App den Start beim Neustart des Handys zu erlauben, die Autostart Berechtigung manuell setzen. Wir können das nicht überprüfen.</string>
+  <string name="permission_todo_autostart">4. Autostart erlauben</string>
+  <string name="send_detected_notif">Benachrichtigung senden</string>
+  <string name="skip">Überspringen</string>
+  <string name="finish">Beenden</string>
+  <string name="okay">Ok</string>
+  <string name="skip_warning">Um sicherzustellen dass diese App korrekt funktionert, musst du diese Berechtigungen erteilen. Du kannst dieses Menü über den \"Berechtigungen\"-Knopf im Home-Menü erneut öffnen.</string>
+  <string name="notice">Hinweis</string>
+  <string name="test_notification_content">Dein Code ist: 230914</string>
+  <string name="test_notification_title">Dies ist eine Testbenachrichtigung</string>
+  <string name="permission_todo_allow_restricted_settings">- Eingeschränkte Einstellungen erlauben</string>
+  <string name="failed_to_open_app_settings">Einstellungen öffnen fehlgeschlagen</string>
+  <string name="perm_restricted_desc">Falls du \"Benachrichtigungen lesen\" in Schritt 2 nicht erlauben konntest, versuche \"Eingeschränkte Einstellungen erlauben\" in den App Einstellungen</string>
+  <string name="en" translatable="false">English</string>
+  <string name="fa" translatable="false">فارسی</string>
```

Comment:
add language code "de" entry under here

Reply:
(in general `translatable="false"` doesn't belong outside the main strings.xml but I will move it after merge, so np)

Reply:
oh yup that makes sense, thanks for the feedback